safe/gui/tools/wizard/step_kw20_unit.py

```diff
@@ -59,7 +60,10 @@ def get_next_step(self):
             new_step = self.parent.step_kw_classification
         else:  # No classifications
             if is_raster_layer(self.parent.layer):
-                new_step = self.parent.step_kw_source
+                if subcategory == exposure_population:
```

Comment:
No. Based on our discussion, we will put this kind of issue in this ticket: https://github.com/inasafe/inasafe/issues/3423